cmd/eks-node-viewer/main.go

```diff
@@ -123,6 +123,11 @@ func startMonitor(ctx context.Context, settings *monitorSettings) {
 				p := obj.(*v1.Pod)
 				if !isTerminalPod(p) {
 					cluster.AddPod(model.NewPod(p), settings.pricing)
+					node, ok := cluster.GetNode(p.Spec.NodeName)
+					// need to potentially update node price as we need the fargate pod in order to figure out the cost
+					if ok && node.IsFargate() && node.Price != node.Price {
```

Comment:
is `node.Price != node.Price` right? Seems like that should always be false?

Reply:
I use NaN to represent an unknown price, and NaN != NaN. Probably not the clearest choice I ever made 😆

Reply:
https://go.dev/play/p/rwq-A_4qDPm